docs/data/data-grid/editing/LinkedFieldsCellEditing.tsx

```diff
@@ -56,7 +56,7 @@ const CustomTypeEditComponent = (props: CustomTypeEditComponentProps) => {
 };
 
 export default function LinkedFieldsCellEditing() {
-  const editingRow = React.useRef<GridRowModel>(null);
+  const editingRow = React.useRef<GridRowModel | null>(null);
```

Comment:
Change in `GridRowModel` exposed a hidden bug in this example (`null` wasn't properly handled)